volume/ceph/ceph.go

```diff
@@ -177,24 +186,64 @@ func (v *Volume) Mount(id string) (mappedDevicePath string, returnedError error)
 	if err != nil {
 		return "", err
 	}
+
+	mappedDevicePath = v.mappedDevicePath
```

Comment:
Fixed in https://github.com/medallia/docker/pull/15/commits/975c9c2ab2e543bd69892b3254de4e3e90c1156e.